root/static/scripts/common.js

```diff
@@ -34,6 +34,7 @@ require('./common/components/CommonsImage');
 require('./common/components/FingerprintTable');
 require('./common/components/WikipediaExtract');
 require('./common/MB/Control/Autocomplete');
+require('./common/components/AttributeList');
```

Comment:
Instead of adding this to every page, can we link it only where it's needed like was done at https://github.com/metabrainz/musicbrainz-server/pull/2309/commits/cd68deecabec27778605f7fe7aaca8758ddaa4e3?

Reply:
Sure. Also updated the PR to do the same to the other `CollapsibleList`s